runtime/src/main/java/org/corfudb/runtime/view/LayoutManagementView.java

```diff
@@ -110,6 +110,38 @@ public void bootstrapNewNode(String endpoint)
         log.info("bootstrapNewNode: New node {} bootstrapped.", endpoint);
     }
 
+    /**
+     * Removes a node from the layout
+     * @param currentLayout the layout to remove the node from
+     * @param endpoint the node to remove
+     */
+    public void removeNode(Layout currentLayout, String endpoint) throws QuorumUnreachableException,
+            CloneNotSupportedException,
```

Comment:
You can catch this where it is being thrown and throw UnrecoverableCorfuException

Reply:
I dont get the point of wrapping all these exceptions in UnrecoverableCorfuException, also there doesn't seem to be a UnrecoverableCorfuException

Reply:
org.corfudb.runtime.exceptions.unrecoverable.UnrecoverableCorfuError? Sorry, not exception.

There is a serious point in wrapping them. Otherwise Exceptions get swallowed by retry logic and the user will never know...

Reply:
And yes, this led to bug #1033